<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IssoDigitalSensorSetupPanel</class>
 <widget class="QGroupBox" name="IssoDigitalSensorSetupPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>688</width>
    <height>64</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>64</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>GroupBox</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
	background-color: rgb(255, 255, 255);
	font-size: 9pt;
}

QSpinBox
{
	background-color: rgb(230, 230, 230);
	border-radius: 6px;
	border: 1px solid gray;
	min-height: 20px;
	min-width: 60px;	
	max-width: 100px;	
}


QComboBox
{
	background-color: rgb(230, 230, 230);
	border-radius: 6px;
	border: 1px solid gray;
	min-width: 60px;
	min-height: 18px;
}
</string>
  </property>
  <property name="title">
   <string>Заголовок</string>
  </property>
  <property name="checkable">
   <bool>true</bool>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>10</number>
   </property>
   <property name="leftMargin">
    <number>20</number>
   </property>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Внимание</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="sbWarning">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="maximum">
      <number>65535</number>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::MinimumExpanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Тревога</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="sbAlarm">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="maximum">
      <number>65535</number>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::MinimumExpanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Условие</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="cbCondition"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
